Libs/RaptureCore/src/main/java/rapture/kernel/DocApiImpl.java

```diff
@@ -786,6 +792,27 @@ public void runIndex(CallingContext context, IndexScriptPair indexScriptPair, St
             if (notEmpty.contains(uri)) continue;
             deleteDoc(context, uri);
         }
+        
+        if (recurse) {
+            String auth = new RaptureURI(docUri).getAuthority();
+            int len = auth.length() + 4 + Scheme.DOCUMENT.toString().length();
```

Comment:
Skipping the scheme and authority using substring. The problem is that listDocsByUriPrefix expects the scheme and authority but repository.removeChildren expects just the doc path.